contracts_and_coroutines.html

```diff
@@ -277,23 +278,26 @@ <h2> Precondition and postcondition properties </h2>
 <h2> Preconditions on coroutines </h2>
 
 
-<p> Preconditions on coroutines are quite intuitive, as the call to a coroutine is 
-    sequential: we ave the point where the caller initializes function parameters,
+<p> Preconditions on coroutines are quite intuitive since the call to a coroutine
+    is sequential: we have the point where the caller initializes function parameters,
     is suspended and the callee starts.
     </p>
     
-<p> The unusual things that happen when a coroutine is started include:</p>
+<p> However, in contrast to a regular function, we specify additional actions
+    that take place when a coroutine is started, specifically:</p>
 
 <ol>
-  <li> Possible allocation of a coroutine frame <a href="http://eel.is/c++draft/dcl.fct.def.coroutine#9">[dcl.fct.def.coroutine]/9</a>
-       which might involve a user-supplied allocator that will read the value of coroutine parameters.</li>
-  <li> Moving of the coroutine parameters into the coroutine frame.</li>
-  <li> The initialization of the coroutine promise object, which may pass those copies as arguments to
-       the constructor of the promise object) <a href="http://eel.is/c++draft/dcl.fct.def.coroutine#5.7">[dcl.fct.def.coroutine]/5.7</a>.</li>
+  <li> Possible allocation of a coroutine state frame <a href="http://eel.is/c++draft/dcl.fct.def.coroutine#9">[dcl.fct.def.coroutine]/9</a>
+       which might involve a user-supplied allocator that is permitted to read
+       the values of coroutine parameters <em>as passed</em>.</li>
+  <li> Copying of the coroutine parameters into the coroutine state frame.</li>

```

Comment:
Again, I will insist that this is actually moving.

Reply:
then we might say "potentially-moved-from" ?

Reply:
Works for me.